recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
When I last tried it, ``/opt/timesync`` belonged to user timesync, but ``/opt/timesync/source`` belonged to root. Maybe because the checkout is being performed by root? I'm not sure why.